index.bs

```diff
@@ -2344,7 +2344,7 @@ Council Composition</h5>
 	Each [=W3C Council=] is composed of the following members (excepting any [[#council-participation|renounced or dismissed]]):
 	* the [=CEO=] or their chosen [=Team=] delegate
 	* the members of the [=Advisory Board=]
-	* the members of the [=Technical Architecture Group=]
+	* the members of the [=Technical Architecture Group=] other than Tim Berners-Lee
```

Comment:
If we take #791, then an alternative solution here would be:

```suggestion
	* the members of the [=Technical Architecture Group=]
	including invitees who have accepted their invitation to participate in this activity
```

Reply:
If we take #791, we don't need this PR at all: they are alternatives to each other.